examples/integration/xcodeproj_targets.bzl

```diff
@@ -18,13 +18,13 @@ XCODEPROJ_TARGETS = [
     ),
     "//iMessageApp",
     top_level_target(
-        label = "//iOSApp",
+        label = "//iOSApp/Source:iOSApp",
         target_environments = ["device", "simulator"],
     ),
     "//macOSApp/Source:macOSApp",
     "//macOSApp/Test/UITests:macOSAppUITests",
     top_level_target(
-        label = "//tvOSApp",
+        label = "//tvOSApp/Source:tvOSApp",
```

Comment:
These changes aren't needed anymore, right?

Reply:
Correct, reverted this.